src/client/dfuse/dfuse.h

```diff
@@ -56,6 +56,11 @@ struct dfuse_info {
 	/* Array of dfuse_eq */
 	struct dfuse_eq     *di_eqt;
 	ATOMIC uint64_t      di_eqt_idx;
+
+	ATOMIC uint64_t      di_inode_count;
+	ATOMIC uint64_t      di_fh_count;
+	ATOMIC uint64_t      di_pool_count;
+	ATOMIC uint64_t      di_container_count;
```

Comment:
i don't think memquery really associate with these counters. a user would think this tool is actually printing memory usage but those counters. so the tool name probably makes more sense to be called stat or probably just query